Client/src/Components/DataCell/DataCell.js

```diff
@@ -64,6 +66,11 @@ export default function DataCell() {
         const i = filename.lastIndexOf('.')
         return (i < 0) ? '' : filename.substr(i)
     }
+    //to do: refactor into proper txt file
+    const readme= "To properly upload a JSON file based on the provided empty sample, \nthe user must fill up the following fields: \n"
+    const fields="-reference \n-type \n-publisher \n-author \n-title \n-year \n-dataset/materials \n-variables/points \n"

```

Comment:
It should be mentioned that volume, pages, datePublished, dateAccessed, details {of materials}, repr, units, subcategory, and data type are optional fields whose lines can/should be removed if not applicable.

Reply:
added it on the latest commit. thanks for the feedback